catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -437,36 +438,38 @@ private static boolean isAttributeMultiValued(String attribute, MetacardImpl met
         return multiValued;
     }
 
-    private static String getPhoneNumberString(TelephoneNumberType digits) {
-        String phoneNumber;
-        if (StringUtils.isBlank(digits.getExtension())) {
-            phoneNumber = String.format("(%s) %s", digits.getAreaCode(), digits.getNumber());
-        } else {
-            phoneNumber = String.format("(%s) %s extension %s",
-                    digits.getAreaCode(),
-                    digits.getNumber(),
-                    digits.getExtension());
-        }
-
-        return phoneNumber;
+    private static String getPhoneNumber(TelephoneNumberType digits) {
```

Comment:
That works for me. 

I'm assuming the area code/prefix/number/extension parts are pretty universal.